cpp/bridge.cpp

```diff
@@ -367,27 +367,30 @@ namespace opsqlite {
                             }
                             i++;
                         }
-                        results->push_back(row);
+                        if (results != nullptr) {
```

Comment:
This is not related to the hooks changes right? Why change it? the `results` vector and the `metadatas` should always be there if I recall correctly

Reply:
If you really want to check for the null pointer then I suggest you move to the top of the function and throw an exception there. I prefer the early return/fail pattern than to silently do nothing.

Reply:
https://github.com/OP-Engineering/op-sqlite/blob/1c0cef742d117cd67aae6a80760cbf441d074901/cpp/sqlbatchexecutor.cpp#L63-L64

In `sqlbatchexecutor` function, it call sqliteExecute with nullptr results and metadatas, I think it is intentional ignoring them.

Reply:
Hmmm I see... maybe we should change it, batch executor instead of using the normal execute, use the executeLiteral one